tests/e2e/index_hints/index_hints.py

```diff
@@ -407,14 +409,15 @@ def test_multiple_match_query(memgraph):
     memgraph.execute("CREATE INDEX ON :Label2;")
     memgraph.execute("CREATE INDEX ON :Label3;")
 
+    # TODO: Fix this test since it has the filtering info wrong (filtering by label that's already indexed)
```

Comment:
Shouldn't we fix that before merging?

Reply:
The correct Filter would be:
```
Filter(m:Label3) right? for the line changed
```

Reply:
I agree it should be fixed but I assessed the effort and it's not low effort, it may take a couple of days to go through the filtering logic so I'd first merge this as this printing is totally wrong.